config/settings.yml

```diff
@@ -1,4 +1,14 @@
-  dor_services:
-    url:  'https://dor-services-test.stanford.test'
-    token: secret-token
-    token_header: 'X-Auth'
+dor_services:
+  url:  'https://dor-services-test.stanford.test'
+  token: secret-token
+  token_header: 'X-Auth'
+
+repository_manager:
+  email: bergeraj@stanford.edu
```

Comment:
I doubt @andrewjbtw  wants this in a public repo.  Maybe make part of shared_configs?   Also, it would be better to get an indirect mailing list set up so it goes with the job, not the person.

Reply:
Yeah, I'd prefer not to have email address in the public code if possible.

Reply:
@ndushay I don't know how to do that. Do you?

Reply:
i think you just add the value to https://github.com/sul-dlss/shared_configs/blob/common-accessioning-prod/config/settings/production.yml.    You can test it locally by adding it to config/settings/development.yml .

Reply:
@ndushay I mean how do you "get an indirect mailing list set up" how does one get a list made?

Reply:
I think maybe one can be requested at lists.stanford.edu?

Reply:
we could use dlss-infrastructure

Reply:
or @andrewjbtw is there an email for the sdr manager as a list?